tests/frontmatter.test.js

```diff
@@ -0,0 +1,28 @@
+const glob = require('glob');
+const matter = require('gray-matter');
+
+describe('frontmatter tests', () => {
+  for (const mdFile of glob.sync('src/content/**/*.md')) {
+    let fm = matter.read(mdFile).data;
+
+    // Ignore the homepage for title and description as it's special.
+    if (!mdFile.match(/index\.md/)) {
+      it(`${mdFile} should have description under 70 chars`, () => {
+        expect(fm.title).toBeDefined();
+        expect(fm.title.length).toBeLessThanOrEqual(70);
+      });
+
+      // Skipping to start with because we don't have this yet.
```

Comment:
Then maybe remove it?

Reply:
Yep, fair enough - I'll add this example to the issue for adding descriptions as a guide.